README.md

```diff
@@ -54,19 +54,45 @@ Browsers don't have native support for Grist [yet :-)] but you can make a little
 </html>
 ```
 
-  * I've been pushing Grist code to https://grist-static.com/ as a CDN; you can produce it all yourself using this repo.
+  * The code in this repository is available from https://grist-static.com/ as a CDN; you can produce it all yourself using this repo.
   * After that, it is just a case of putting a `.grist` file on your server beside this `.html` file, and filing in the options to `bootstrapGrist`.
   * You can also pass `initialData: 'path/to/data.csv'` to import a CSV file into a new table. In this case `initialFile` is optional.
   * There is also `initialContent` option. You can use it to pass the content of a CSV file.
   * You can also pass `elementId: 'element-id` to open Grist within an element in your page.
 	- In that case, you can include a small wrapper page for your document as above, and embed it as an iframe yourself.
   * You can set `singlePage: true` for a less busy, single page layout.
 
+## Grist CSV Viewer
 
-### No-Code integration
+Grist can handle data in multiple formats, including CSV. Since CSV is such a common format, and interacting with it
+online remains a chore, we've packaged `grist-static` into a streamlined `csv-viewer.js` utility specifically for
+viewing, sorting, filtering, any copy/pasting from CSV, directly in a webpage.
 
-We have a few building blocks ready for you, that makes **embedding** CSV or Grist files as easy as possible.
-In order to use them, first include the `csv-viewer.js` script somewhere near the top of the page. The first one is a custom `csv-viewer` web element.
+Just add the viewer in the `head` area of a webpage:
+
+```html
+<head>
+  <script src="https://grist-static.com/csv-viewer.js"></script>
+</head>
+```
+
+Then you can make a button to open CSV from a URL:
+```html
+<button data-grist-csv-open="transactions.csv">View CSV</button>
+```
+The CSV could be a file, or something your site generates for a user dynamically.
```

Comment:
If "something" has to be a URL, I think better to be explicit, i.e. "or a URL of CSV data that your site generates for a user dynamically".